src/components/LocksCard.tsx

```diff
@@ -129,96 +128,90 @@ export const LocksCard = () => {
-            </div>
-            {freeLocks.length > 0 && (
-              <>
-                <Button
-                  className="mb-2 mt-4 w-full"
-                  onClick={onUnlockClick}
-                  disabled={isUnlockingLoading}
-                >
-                  Unlock
-                </Button>
-                <ContentReveal hidden={false}>
-                  {freeLocks.map((lock) => {
-                    if (lock.type === LockType.Delegating) {
-                      const { amount, trackId } = lock
-                      return (
-                        <div key={trackId}>
-                          <ul>
-                            <li className="mb-2">
-                              <div className="capitalize">
-                                <TrackDisplay trackId={trackId} />
-                                <div>
-                                  <BadgeCent className="inline-block h-4 w-4 text-gray-500" />{' '}
-                                  {planckToUnit(
-                                    amount,
-                                    assetInfo.precision,
-                                  ).toLocaleString('en')}{' '}
-                                  {assetInfo.symbol}
-                                </div>
-                              </div>
-                            </li>
-                          </ul>
+    <div className="flex flex-col gap-2 md:grid md:grid-cols-3 md:flex-row">
+      <Card className="h-max border-2 p-2 px-4">
+        <div className="relative z-10">
+          <div className="flex gap-x-2">
+            <Title variant="h4">Unlockable</Title>
+            <Popover>
+              <PopoverTrigger>
+                <Info className="h-3 w-3 text-gray-500" />
+              </PopoverTrigger>
+              <PopoverContent>
+                <p className="max-w-[15rem]">Funds that are unlockable.</p>
+              </PopoverContent>
+            </Popover>
+          </div>
+          <div className="font-unbounded text-5xl font-bold">
+            {freeLocks.length}
+            <LockKeyholeOpen className="ml-1 inline-block h-8 w-8 rotate-[10deg] text-gray-200" />
+          </div>
+          {freeLocks.length > 0 && (
+            <Button
+              className="mb-2 mt-4 w-full"
+              onClick={onUnlockClick}
+              disabled={isUnlockingLoading}
+            >
+              Unlock
+            </Button>
+          )}
+          <ContentReveal hidden={freeLocks.length == 0}>
+            {freeLocks.map((lock) => {
+              if (lock.type === LockType.Delegating) {
+                const { amount, trackId } = lock
+                return (
+                  <div key={trackId}>
+                    <ul>
+                      <li className="mb-2">
+                        <div className="capitalize">
+                          <span className="capitalize">
+                            <Badge>{trackList[trackId]}</Badge> /{trackId}
```

Comment:
Conflict went wrong here, it should be
```suggestion
                            <TrackDisplay trackId={trackId} />
```